src/interface_adapter/statistics/StatisticsState.java

```diff
@@ -0,0 +1,6 @@
+package interface_adapter.statistics;
+
+public class StatisticsState {
+    public String stats = "";
```

Comment:
Possibly not a big issue, but instead of just having one singular string for the state it may be more intuitive if we differentiate the stats. For example, have a numGamesPlayed, avgScore, etc attributes. Then in the pop up that displays the stats you can call the getter methods for these attributes and set the text that way. What are your thoughts @rayceramsay? 

Also note, if this is something we want to change, the output data file will also need to be updated accordingly. Just wanted to add that as a note.